File: Webfront/src/modules/intern-eggrolls/components/Model/Answer.ts
export class Answer{
    private Ans:string;
    constructor(tmpAns = ""){
        this.Ans = tmpAns;
    }

    get ans() : string{
        return this.Ans;
    }
    set ans(tmpAns : string){
        this.Ans = tmpAns;
    }
}

File: back-end/01_Resources/Common/EggRolls/Entities/Answer.cs
namespace Newegg.MIS.API.EggRolls.Entities
{
    public class Answer : Participator
    {
        public int? QuestionnaireID { get; set; }
        public int? TopicID { set; get; }
        public string Ans { set; get; }
        public TopicType? Type { get; set; }
    }
}
